src/styles/theme.scss

```diff
@@ -142,15 +160,16 @@ h5 {
 }
 
 .link--plain {
-  color: $interaction-colour;
+  color: $text-colour;
```

Comment:
links should maybe be another colour?

Reply:
following up the design file but will check again tomorrow with fresh eyes - would be good for Holly or Rose to offer an expert eye on this. My sense of colour is horrendous lol

![Screenshot from 2020-01-09 18-07-31](https://user-images.githubusercontent.com/29477363/72092781-f1583200-330a-11ea-83d9-941513ea8126.png)